templates/toil-instance.yaml

```diff
@@ -323,14 +334,16 @@ Resources:
               owner: "root"
               group: "root"
           commands:
-            01_enable_toil-cluster-service:
-              command: "/bin/systemctl enable toil-cluster.service"
-            02_start_toil-cluster-service:
-              command: "/bin/systemctl start toil-cluster.service"
-        ReloadCluster:
-          commands:
-            01_reload_toil-cluster-service:
-              command: "/bin/systemctl reload toil-cluster.service"
+            01_update-rc.d_toil-cluster-service:
```

Comment:
i assume this is to add/install the service?  maybe name here it something like `01_install-toil-cluster-service`?

Reply:
This is to add the service to the runlevels so that sysv knows when to invoke it. I will rename to `01_add_toil-cluster-service_run-levels`.

Reply:
Actually, I will also add a comment there about runlevels in case people are not familiar with the antique ways of sysv.